.github/workflows/prettier.yaml

```diff
@@ -0,0 +1,26 @@
+name: Apply Prettier style
+
+on:
+  push:
+    paths:
+      - "**.md"
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20.x"
+      - run: npm install --no-save prettier@3
+      - run: npx prettier --write .
+      - name: git config
+        run: |
+          git config user.name 'github-action'
```

Comment:
maybe it should be `actions` because https://github.com/actions

Reply:
Ah, wait: Did i misunderstand, and you meant that the name should be `actions` rather than `github-actions`?